mergify_engine/tests/unit/test_debug.py

```diff
@@ -0,0 +1,44 @@
+# -*- encoding: utf-8 -*-
+#
+# Copyright © 2021 Mergify SAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import typing
+
+import pytest
+
+from mergify_engine import debug
+
+
+res = ("mergifyio", "mergify-engine", "123")
```

Comment:
You only have one result here for all test cases, so no need to pass it as an argument to `test_parser`.